packages/airnode-deployer/terraform/airnode/aws/variables.tf

```diff
@@ -41,5 +41,6 @@ variable "api_key" {
 
 variable "api_max_concurrency" {
   description = "Maximum amount of concurrent executions for Airnode Test Gateway Lambda"
+  # https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function#reserved_concurrent_executions
```

Comment:
That link only mentions AWS, but we also want this parameter for GCP?

Reply:
That's in the `gcp` directory. I added the link for AWS because of the non-intuitive -1. I think that's enough. Otherwise I could add links to all the parameters.